primitives/src/policy.rs

```diff
@@ -136,6 +136,9 @@ impl Policy {
     /// 25 MB.
     pub const HISTORY_CHUNKS_MAX_SIZE: u64 = 25 * 1024 * 1024;
 
+    /// The PoW genesis timestamp.
+    pub const POW_GENESIS_TIMESTAMP: u64 = 1523727000;
```

Comment:
What's the unit here? It looks like a timestamp using seconds as resolution, but I think we need milliseconds.

Reply:
Yeah, you're right